content/guidance/financial-support-for-teacher-training.md

```diff
@@ -6,6 +6,11 @@ related_content:
   Teacher training funding: https://www.gov.uk/teacher-training-funding
   "Early years initial teacher training: 2020 to 2021 funding guidance": https://www.gov.uk/guidance/early-years-initial-teacher-training-2020-to-2021-funding-guidance
   "Qualified teacher status (QTS): qualify to teach in England": https://www.gov.uk/guidance/qualified-teacher-status-qts
+date: "2021-02-26"
+description: |-
+  There are several types of funding available to help you become a teacher, from
```

Comment:
Maybe we could use a bit more 'sell'y language here like: 

"Become a teacher with the help of a bursary, scholarship or loan. Extra help is available is you're disabled or if you're a parent or carer."